dbms/src/Storages/DeltaMerge/DeltaMergeStore_Ingest.cpp

```diff
@@ -1155,7 +1154,8 @@ Segments DeltaMergeStore::buildSegmentsFromCheckpointInfo(
         return {};
     }
     wbs.writeLogAndData();
-    LOG_INFO(log, "Finish write fap checkpoint, region_id={}", checkpoint_info->region_id);
+    wbs.writeRemoves();
```

Comment:
It is originally in tiflash/dbms/src/Storages/DeltaMerge/DeltaMergeStore_Ingest.cpp at 821997be4c542acdacb7536c98016963f

Reply:
I didn't get it. This line is still in `DeltaMergeStore::ingestSegmentsFromCheckpointInfo` in your PR. Why we need to call `wbs.writeRemoves()` here and what pages tend to be removed?
https://github.com/pingcap/tiflash/blob/85e144c75266ee40dd0fc5e8720f1405e4344353/dbms/src/Storages/DeltaMerge/DeltaMergeStore_Ingest.cpp#L1206-L1214

Reply:
Fixed